.gitignore

```diff
@@ -15,3 +15,5 @@ npm-debug.log
 
 # Regularly changing or system specific IDE files
 /.idea/workspace.xml
+build/
+*.json
```

Comment:
This would exclude any configuration files which are required to make the project. Please remove this